checker/manual/constant-value-checker.tex

```diff
@@ -185,22 +224,27 @@ \section{Warnings\label{value-checker-warnings}}
 \end{itemize}
 \end{sloppypar}
 
-There is one other situation in which the Constant Value Checker produces a
+There are some other situations in which the Constant Value Checker produces a
 warning message:
 
 \begin{sloppypar}
 \begin{itemize}
-\item \code{[too.many.values.given] Annotation ignored because the maximum
-  number of values tracked is 10.}
+\item \code{[too.many.values.given] The maximum number of arguments permitted is 10.}
 
   The Constant Value Checker only tracks up to 10 possible values for an
   expression.  If you write an annotation with more values than will be
-  tracked, the annotation is ignored.
+  tracked, the annotation is replaced with \<@IntRange> or \<@UnknownVal>.
 
-\end{itemize}
-\end{sloppypar}
+\item \code{[too.narrow.int.range] The smallest interval of integer range permitted is 10.}
+
+  Vice versa.
 
+\item \code{[from.greater.than.to] The integer range specified is abnormal.}
```

Comment:
`from.greater.than.to` should not be a warning.  It should be an error, and there is no need to describe it in the manual.  (Only messages that might be confusing need to be explained.)